src/app/components/our-team/our-team.component.scss

```diff
@@ -150,8 +160,16 @@
       position: absolute;
       width: 100%;
       top: 0;
-      left: 0;
+      left: -15px;
```

Comment:
Can we make it relative to the device width?

Reply:
@Sanji515 is this done?

Reply:
@lunayach I checked it but it was a very little change from the starting position of card so I keep it as it is and It's looking good in small devices also.

Reply:
@lunayach @Shekharrajak done